index.html

```diff
@@ -0,0 +1,272 @@
+<!DOCTYPE html>
+<html lang="en-us">
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="stylesheet" href="./assets/css/style.css" />
+    <title>My Portfolio</title>
```

Comment:
Consider making this more user friendly or more detailed. E.g. 'Abdirahman Osman: Portfolio'